docs/notebooks/figanos_multiplots.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<p>I might change this to just " Multiple Plots"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>